share/management/commands/loadsources.py

```diff
@@ -132,7 +133,11 @@ def schedule_harvest_task(self, label, disabled):
     def process_defaults(self, model, defaults):
         ret = {}
         for k, v in defaults.items():
-            field = model._meta.get_field(k)
+            try:
```

Comment:
This script gets run by a migration before the canonical field is created. This just ignores fields that don't exist when this script is run. I'll add a note.